cmd/state/stateless/state.go

```diff
@@ -340,15 +344,17 @@ beginTx2:
 		goto beginTx2
 	}
 
-	r.LastTimestamps.Walk(func(_ string, lt uint64) bool {
+	if err := r.lastTimestamps.ForEach(func(_ []byte, lt uint64) error {
```

Comment:
It is best you commit `txLocal` before you start iteration - it is more efficient that way